runtime/oti/j9nonbuilder.h

```diff
@@ -140,6 +140,13 @@
 #define J9ArrayShape32Bit 0x2
 #define J9ArrayShape64Bit 0x3
 
+/* J9IndexableObjectLayout_xxx are used in indexableObjectLayout fieldo */
```

Comment:
typo in 'fieldo' (extra 'o' at the end)
and while you are editing it, add '... field in J9VMThread and J9JavaVM'